<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UdpWindow</class>
 <widget class="QMainWindow" name="UdpWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>127</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UdpWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnSend">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>90</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>90</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Host</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Contents</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtHost">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>691</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.0.114</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtPort">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>691</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>9002</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtBody">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>691</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>&quot;{ &quot;data&quot; : { &quot;mesage&quot;:&quot;hello&quot; }, &quot;from&quot; : &quot;{ef40747a-2fa6-11e1-a373-001cc0b9e348}&quot;, &quot;type&quot; : &quot;test&quot; }&quot;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
